README.md

```diff
@@ -25,21 +23,27 @@ npm install svelte-tag
 ```
 
 ```javascript
-import Component from "svelte-tag"
+import SvelteTag from "svelte-tag"
```

Comment:
This was an opinionated change. In my defense, not only did I naturally want to name it  `SvelteTag`, but apparently so have others (see dependents). Seems natural 😊, e.g. https://github.com/souriya777/elegant-seo-website/blob/main/_dev/src/js/svelte-elements.js